packages/manager/apps/dedicated/client/app/license/order/license-order.controller.js

```diff
@@ -231,6 +236,34 @@ angular
             $scope.selected.version =
               $scope.types[$scope.selected.licenseType].options[0].options[0];
           }
+          const { options } = $scope.types[
+            $scope.selected.licenseType
+          ].options[0];
+          if (
```

Comment:
Could be turned into:

```js
if (['CPANEL', 'WORKLIGHT'].includes($scope.selected.licenseType)) {
  // …
}
```

Reply:
since there are only two, I have used || operator.

Reply:
Yes but it will be easier if we have to add another one